packages/city/city.graphql

```diff
@@ -0,0 +1,14 @@
+type City {
+    name: String
+    weatherCode: ID
+    currentWeather: Weather
+}
+
+extend type Weather @key(fields: "code") {
```

Comment:
My IDE is complaining that this directive is not found, similarly complaining in lots of places that stuff is being redefined as a result of the supergraph schema. I wonder if there's a way to make everyone happy in a workspace set up.

Reply:
Might be an IDE specific thing, VSCode with the graphql plugins and it's all good. What ide are you using?

Reply:
intellij IDEA